drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -573,6 +581,11 @@ static void _isr(netdev_t *netdev)
                 at86rf2xx_set_state(dev, dev->idle_state);
                 DEBUG("[at86rf2xx] return to state 0x%x\n", dev->idle_state);
             }
+#ifdef AT86RF2XX_XAH_CTRL_2__ARET_FRAME_RETRIES_OFFSET
```

Comment:
add `/* Only radios with the XAH_CTRL_2 register support frame retry reporting */` here, too